### DISCUSSION TITLE
Ask GitHub copilot

### DISCUSSION THREAD

**Discussion by Biantuadi** [community/community#39267] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Good morning,
Last week I had subscribed for github copilot 10€ per me, and it was last week that I activated github for students and I just learned that copyilot is free for students but yet I was prelifted 10€ today around 8am.
I tried to stop the subscription for me everything was okey; but yet I just received an email with a fee of 80€....
I don’t understand why.
Since yesterday I created the issue... But until GitHub doesn't respond me.
Can GitHub refund me?
thank you.

**Comment by jfullstackdev** [comment, marked as answer]
as far as I know, it's only free for students with the Student Developer Pack

![image](https://user-images.githubusercontent.com/47092464/204072222-43b3286d-7e78-42a4-aeec-68c0ab6d72b8.png)

without that, it will be charging you
